test/test_matplotlib.py

```diff
@@ -16,3 +16,15 @@ def test_svg(selenium):
     content = selenium.run("fd.getvalue().decode('utf8')")
     assert len(content) == 15752
     assert content.startswith("<?xml")
+
+
+def test_pdf(selenium):
+    selenium.load_package("matplotlib")
+    selenium.run("from matplotlib import pyplot as plt")
+    selenium.run("plt.figure()")
+    selenium.run("x = plt.plot([1,2,3])")
+    selenium.run("import io")
+    selenium.run("fd = io.BytesIO()")
+    selenium.run("plt.savefig(fd, format='svg')")
```

Comment:
Shouldn't the format be ``pdf`` instead of ``svg`` ?

Reply:
Good catch!  Thanks.